docs/source/scaleVM.md

```diff
@@ -0,0 +1,163 @@
+| `WINDOWS_URL` | URL to the Windows .qcow2 image shared via NGINX (e.g., `http://<host>:8083/windows.qcow2`) |
+
+### Example Script
+
+```bash
+#!/bin/bash
+
+# Load Windows VMs including verification (verify login home page)
+sudo podman run --rm -t \
+  -e WORKLOAD='windows_vm' \
+  -e KUBEADMIN_PASSWORD="${KUBEADMIN_PASSWORD}" \
+  -e SAVE_ARTIFACTS_LOCAL=True \
+  -e SCALE="${WINDOWS_SCALE}" \
+  -e SCALE_NODES="${SCALE_NODES}" \
+  -e THREADS_LIMIT="${THREADS_LIMIT}" \
+  -e WINDOWS_URL="${WINDOWS_URL}" \
+  -e RUN_STRATEGY=True \
+  -e TIMEOUT="${TIMEOUT}" \
+  -e log_level="INFO" \
+  -v "${KUBECONFIG_PATH}:${KUBECONFIG_PATH}" \
+  -v /tmp/benchmark-runner-run-artifacts:/tmp/benchmark-runner-run-artifacts \
+  --privileged quay.io/ebattat/benchmark-runner:latest
+
+```
+
+# 📊 Optional: Elasticsearch Integration for Metrics Collection
+
+This document explains how to integrate **Elasticsearch** into your VM scaling process to collect and store performance metrics, including **bootstorm time** and `virtctl` access status. 
+
+---
+
+## 🚀 Features
+
+- Collects performance metrics like **bootstorm time** (the time it takes for a VM to boot and become reachable) and `virtctl_vm` status.
+- Optional Elasticsearch integration with the benchmark runner.
+- Allows central storage and analysis of metrics for better performance insights.
+
+---
+
+## 🧪 Prerequisites
+
+- A running **Elasticsearch** instance (can be self-hosted or cloud-based).
+- Proper access to Elasticsearch from the system running the benchmark runner.
+- Ensure that the `ELASTICSEARCH` and `ELASTICSEARCH_PORT` environment variables are configured correctly in your benchmarking scripts.
+
+---
+
+## ⚙️ Parameters for Elasticsearch
+
+| Variable | Description |
+|----------|-------------|
+| `ELASTICSEARCH` | Hostname or IP address of the Elasticsearch server (e.g., `http://localhost`) |
+| `ELASTICSEARCH_PORT` | Port on which Elasticsearch is running (e.g., `9200`) |
+
+---
+
+## 🧰 Integrating Elasticsearch with Benchmark Runner
+
+To enable Elasticsearch integration, simply pass the following environment variables when running the benchmark runner container:
+
+### Example Script with Elasticsearch Integration
+
+```bash
+#!/bin/bash
+
+# Example run with Elasticsearch integration for metrics collection
+sudo podman run --rm -t \
+  -e WORKLOAD='windows_vm' \
+  -e KUBEADMIN_PASSWORD="${KUBEADMIN_PASSWORD}" \
+  -e SAVE_ARTIFACTS_LOCAL=True \
+  -e SCALE="${WINDOWS_SCALE}" \
+  -e SCALE_NODES="${SCALE_NODES}" \
+  -e THREADS_LIMIT="${THREADS_LIMIT}" \
+  -e WINDOWS_URL="${WINDOWS_URL}" \
+  -e RUN_STRATEGY=True \
+  -e TIMEOUT="${TIMEOUT}" \
+  -e log_level="INFO" \
+  -e ELASTICSEARCH="${ELASTICSEARCH}" \
+  -e ELASTICSEARCH_PORT="${ELASTICSEARCH_PORT}" \
+  -v "${KUBECONFIG_PATH}:${KUBECONFIG_PATH}" \
+  -v /tmp/benchmark-runner-run-artifacts:/tmp/benchmark-runner-run-artifacts \
+  --privileged quay.io/ebattat/benchmark-runner:latest
```

Comment:
Same, see if it can be done w/o privilege.

Reply:
removed privilege